src/server/migrations/20200615164201_game_scores.js

```diff
@@ -0,0 +1,15 @@
+exports.up = function (knex) {
+  return knex.schema.createTable('game_scores', (table) => {
+    table.increments('id');
+    table.integer('score');
+    table.integer('fk_user_id');
+    table.foreign('fk_user_id').references('id').inTable('users');
+    table.datetime('created_at').defaultTo(knex.fn.now()).notNullable();
+    table.datetime('updated_at').defaultTo(knex.fn.now()).notNullable();
```

Comment:
For a reference, in this [PR](https://github.com/HackYourFuture-CPH/fp-class12/pull/119) we modified the tables with dates to be timestamp. So consider to wait until that PR is merge, update the branch with develop and then merge this one :)